README.rst

```diff
@@ -106,18 +106,20 @@ There are several ways to specify a database number:
 - If using the ``redis://`` scheme, the path argument of the URL, e.g.
   ``redis://localhost/0``
 
-When using `Redis' ACLs <https://redis.io/topics/acl>`_, you will need to add the
-username to the URL (and provide the password with the Cache ``OPTIONS``).
+When using `Redis' ACLs <https://redis.io/topics/acl>`_, you can either pass the
+username and password in the Cache ``OPTIONS``, or in the URL.
+
 The login for the user ``django`` would look like this:
 
 .. code-block:: python
 
     CACHES = {
         "default": {
             "BACKEND": "django_redis.cache.RedisCache",
-            "LOCATION": "redis://django@localhost:6379/0",
```

Comment:
I would keep both examples if they work

Reply:
I merged the example in which the username is passed in the URL and the password in the OPTIONS with the example of a URL unsafe password, because I thought it was redundant. Tell me if it suits you as done in [039432c](https://github.com/jazzband/django-redis/pull/657/commits/039432cd21748aa79dcc32541b657c289f74c306)